app/javascript/packs/runner_application.js

```diff
@@ -19,7 +19,36 @@ Array.prototype.forEach.call(elements, function(element) {
   });
 });
 
+// the autocomplete component only updates the underlying <select> element when
+// a valid option is chosen. If a user sets an answer, then returns to change
+// their answer and either blanks the autocomplete or enters an invalid/partial string
+// the submitted value will not be changed. This is strange UX and confusing for
+// the user.
+// On autocomoplete pages, on submit, we compare the value in the autocomplete
+// input with the underlying select* if they are different we set the selects
+// value to empty in order to trigger validation
+// * the autocomplete value will just be text, the selects value will be a json
+// string {'text': 'United Kingdom', value: 'UK'}
+
+const autocompleteComponent = document.querySelector('[data-fb-content-type="autocomplete"]');
+if(autocompleteComponent) {
+  const autocompleteForm = autocompleteComponent.parentNode;
+
+  autocompleteForm.addEventListener('submit', function(event) {
+    const form = event.target;
+    const autocompleteField = form.querySelector('input.autocomplete__input');
+    const autocompleteSelect = form.querySelector('.fb-autocomplete');
+    
+    // if the select is not empty and the values do not match or if the
+    // autocomplete is empty trigger validation
+    if(autocompleteSelect.value != '' && !autocompleteSelect.value.includes(autocompleteField.value) || autocompleteField.value == '') {
+      autocompleteSelect.value = '';
+    }
+  });
+}
+
```

Comment:
One last thing, for which I understand there maybe a perfectly valid reason since it is on the Runner, you're stuffing JS code into the sites JS manifest file. 

The Editor counterpart file start with this:
```
// This file is automatically compiled by Webpack, along with any other files
// present in this directory. You're encouraged to place your actual application logic in
// a relevant structure within app/javascript and only use these pack files to reference
// that code so it'll be compiled.


// Scripts called by javascript_pack_tag
require("jquery")
require("jquery-ui")
require("showdown")
require("../src/index")
```

I don't think I wrote that opening comment (Guessing Rails standard) but it does tell you what is 'wrong' in approach here. This code should be in a site 'index' file (if global and on each page) or imported from something more functionality specific.
e.g.
`require('./src/autocomplete_support');`

For what it's worth, and taking in other comments above, I wouldn't recommend having/needing an 'autocomplete_support' file. 

I'd personally expect this Runner manifest file to look more like:

```
require("../src/analytics")
require("../src/index")
```

With the inside of index.js (or some other organised set of files) to look something like:

```
const accessibleAutocomplete = require("accessible-autocomplete");

function init() {
  createAutoCompleteComponents();
}

// Find and create Autocomplete components
function createAutoCompleteComponents() {
    var elements = document.querySelectorAll('[data-fb-content-type="autocomplete"]');
    elements.forEach(function(element) ){
        new AutoCompleteThingy(element, {
                // blah config
        });
    });
}
```